.ci/job_matrix.yaml

```diff
@@ -99,8 +99,9 @@ steps:
   - name: Run UCC MPI tests
     agentSelector: "{nodeLabel: 'swx-clx01'}"
     run: |
+      [ "$UCC_MPI_TESTS" = "false" ] && { echo "MPI tests were skipped !!!";exit 0; }
```

Comment:
{ echo "MPI tests were skipped !!!";exit 0; }
->
{ echo "WARNING: MPI tests were skipped !!!"; exit 0; }

Reply:
Or even to:
if [[ "$UCC_MPI_TESTS" == "false" ]]; then
    echo "WARNING: MPI tests were skipped !!!"
    exit 0
fi